docs/reference_guide.md

```diff
@@ -3255,6 +3256,32 @@ Attaching 1 probe...
 Operation not permitted
 ```
 
+# 35. `offsetof`: Offset of element in structure
+
+Syntax: `offsetof(struct, element)`
+
+Get the offset of the element in the struct.
+
+Examples:
+
+```
+#!/usr/bin/env bpftrace
+#include <linux/sched.h>
```

Comment:
Recommend dropping the `#include`. Most linux distros ship with BTF built in now; bpftrace can automatically extract all necessary definitions from it.